src/main/java/com/epam/reportportal/annotations/Step.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.reportportal.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
```

Comment:
Please, give documentation for this annotation

Reply:
added

Reply:
done